sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -186,6 +194,20 @@ public void testAlterTableWithRefreshInterval() throws Exception {
     }
 
     @Test
+    public void testAlterTableWithTotalFieldsLimit() throws Exception {
+        AlterTableAnalyzedStatement analysisSet = e.analyze(
```

Comment:
oops, of course it doesn't make sense to outsource the reset to default function since the setting needs to be changed from default before that